package.json

```diff
@@ -32,6 +32,7 @@
         "type-check:force": "rimraf -rf -- **/libDev && yarn type-check",
         "test:unit": "yarn nx run-many --target=test:unit",
         "lint:js": "eslint . --cache --cache-strategy content --flag unstable_config_lookup_from_file --max-warnings 0",
+        "lint:js:fix-files": "eslint --cache --cache-strategy content --flag unstable_config_lookup_from_file --max-warnings 0 --fix",
```

Comment:
nit: you could name it just `lint:js:fix`.
Because it doesn't need files arguments, it works on its own, and that's good - previously I used `yarn lint:js --fix` a lot :+1:

Reply:
for me it has been very desirable :laughing: 
So I thought this new script can save me time. But now I'm thinking about it again - with the commit hook I won't ever need to `yarn lint:js --fix`, as I have been doing past few weeks.
So ignore this.